internal/app/vodtester/vodtester_app.go

```diff
@@ -144,11 +145,107 @@ func (vt *vodTester) Start(fileUrl string, taskPollDuration time.Duration) (int,
 		}
 	}
 
+	filePath := "/tmp/uploadtester.mp4"
+	resp, err := http.Get(fileUrl)
```

Comment:
As we chatted on Discord, instead of downloading a new file we should just receive the `fileName` as args to the `Start` function, just like we already have for the `ContinuousRecordTester`: https://github.com/livepeer/stream-tester/blob/adbe3a0cf54c7eb79f61eee092b7080e26720508/cmd/recordtester/recordtester.go#L275

The `ContinuousVodTester` is right below that, and should also receive the `fileName` as arg and pass it down to this `VodTester` here. Then we only need to open the file instead of re-downloading with HTTP.

We also chatted this offline, but FTR: this file is already present in the Docker image as well, so we don't need the extra bandwidth to download it on every execution of the test.